product.html

```diff
@@ -141,5 +142,154 @@
+                  </g>
+                  <defs>
+                    <clipPath id="clip0_2923_446">
+                      <rect width="16" height="16" fill="white"/>
+                    </clipPath>
+                  </defs>
+              </svg> Retiro en Tienda</button>
+            </div>
+          </div>
+          <div class="product__add-card">
+            <div class="product__add-card-counter">
+              <button class="product__add-card-counter-button product__add-card-counter-button--more-add">
```

Comment:
Como el parent seria un formulario, estos botones para sumar y restar cambiarian a ser de type="button"
```suggestion
              <button class="product__add-card-counter-button product__add-card-counter-button--more-add" type="button">
```